atomix/storage/pom.xml

```
@@ -0,0 +1,62 @@
+<!--
+  ~ Copyright 2017-present Open Networking Foundation
+ * Copyright © 2020 camunda services GmbH (info@camunda.com)
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <artifactId>atomix-storage</artifactId>
+
+  <modelVersion>4.0.0</modelVersion>
+  <name>Atomix Storage</name>
+  <packaging>jar</packaging>
+
+  <dependencies>
+    <dependency>
+      <artifactId>atomix-utils</artifactId>
+      <groupId>io.zeebe</groupId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <artifactId>simpleclient</artifactId>
+      <groupId>io.prometheus</groupId>
+      <version>${prometheus.client.version}</version>
```

```suggestion
      <version>${version.prometheus}</version>
```

That said, we probably don't need to specify the version at all right?